quote-selection.js

```diff
@@ -5,6 +5,19 @@ import selectionToMarkdown from './markdown-parsing'
 const containers = new WeakMap()
 let installed = 0
 
+type Subscription = {|
+  unsubscribe: () => void
+|}
+
+export function subscribe(container: Element): Subscription {
```

Comment:
Do you think this could just be the `export default`?

Reply:
It could. Sorry; I've seen your approval but not the subsequent comment.